src/main/target/REVOLT/REVOLTOSD.mk

```diff
@@ -0,0 +1,8 @@
+F405_TARGETS   += $(TARGET)
```

Comment:
This isn't needed. The make file settings will be taken from `target.mk` for the variant as well, and this can be an empty file.